### DISCUSSION TITLE
Flasher With Symfony form validation and redirectToRoute

### DISCUSSION THREAD

**Discussion by xribant** [php-flasher/php-flasher#26] [Q&A]
Hi,

I tried to setup and use php-flasher/flasher-toastr-symfony with Symfony of course and I ran into a problem when I use it from a form validation.

When I use this : 

```php
if($form->isSubmitted() and $form->isValid()) {
    $toastr
        ->success('<strong>Message reçu! <br>Nous y répondrons dans les meilleurs délais.</strong>')
        ->timeOut(5000)
        ->progressBar()
        ->closeButton()
        ->positionClass('toast-top-left')
        ->flash()
    ;

    return $this->redirectToRoute('home');
}
```

It doesn't works, nothing is sent to the twig template.
When I use the same envelop with a render function, it works fine.

Did I missed something? Do you have any advise?

Thanks in advance.

Xavier.

**Comment by yoeunes** [comment]
Hello @xribant thanks for reporting the issue, I see where the problem is, I'll let you know when it's fixed.

> The messages are stored in an Array adapter and not in the session

**Comment by yoeunes** [comment]
Hello @xribant, can you please upgrade to v0.9.6 and verify it works for you

**Reply by xribant** [reply]
Hello Yoeunes,

You did the trick! It's woking now like a charm.
Thanks for the qui reactivity. Your package is so easy to use and exactly what I need.

Awesome! 

Xavier.

**Reply by yoeunes** [reply]
Great, I'm super glad it fits your needs 😊
